### PR TITLE
Adds Flutter 3.0 breaking changes

### PR DESCRIPTION
**Pull request by MiraMarshall**
_Description of what this PR is changing or adding, and why:_
Adds Flutter 3.0 breaking changes.
_Issues fixed by this PR (if any):_
#7333 
## Presubmit checklist
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/master/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/release/breaking-changes/index.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ release, and listed in alphabetical order:
 [Default PrimaryScrollController on Desktop]: {{site.url}}/release/breaking-changes/primary-scroll-controller-desktop
 [ThemeData's toggleableActiveColor property has been deprecated]: {{site.url}}/release/breaking-changes/toggleable-active-color
 
-### Released in Flutter 3.0
+### Released in Flutter 3
```
Comment (domesticmouse) [inline src/release/breaking-changes/index.md:21]: Will this get confusing once we have "Released in Flutter 3.x"?

Comment (MiraMarshall) [reply src/release/breaking-changes/index.md:21]: I was following the pattern from the Flutter 2 breaking changes. Do you think 3.0 would be better?

Comment (domesticmouse) [reply src/release/breaking-changes/index.md:21]: I suspect @sfshaza2 would prefer Flutter 3 😎

Comment (sfshaza2) [reply src/release/breaking-changes/index.md:21]: Yes, it's "Flutter 3". This is a directive from Tim Sneath.

**Review by sfshaza2 — APPROVED** [review]
lgtm

**Pull request merged**